lib/tasks/csv.rake

```diff
@@ -14,4 +14,22 @@ namespace :csv do
     end
   end
 
+  desc "export all buckets that is funded and has archived_at set as CSV"
```

Comment:
"export all buckets that are funded and have archived_at set to a CSV"

Reply:
@jlopker Thanks 👍